pom.xml

```diff
@@ -139,7 +139,8 @@
     <chill.version>0.9.3</chill.version>
     <ivy.version>2.4.0</ivy.version>
     <oro.version>2.0.8</oro.version>
-    <codahale.metrics.version>3.1.5</codahale.metrics.version>
+    <dropwizard.metrics.version>1.3.10</dropwizard.metrics.version>
+    <codahale.metrics.version>4.0.5</codahale.metrics.version>
```

Comment:
Let's avoid bumping version in the same PR until it is necessary for the functionality. Will the changes work with `3.1.5` of Codahale metrics?

Reply:
There is a class `MetricAttribute` used in `1.3.10` version of dropwizard-metrics. `MetricAttribute` class is part of `4.0.5` version of codahale-metrics. It was not available in `3.1.5` version of codahale-metrics.